content/docs/conditional-rendering.md

```diff
@@ -24,7 +24,7 @@ function GuestGreeting(props) {
 }
 ```
 
-We'll create a `Greeting` component that displays either of these components depending on whether a user is logged in:
+हम `Greeting` कौम्पोनॅन्ट बना रहे है। अगर यूजर लॉगिन है तभी देखेगा:
```

Comment:
हम ek `Greeting` कौम्पोनॅन्ट banaenge jo tabhi dikhega jab user logged in hai

Reply:
done

Reply:
logged in, not log in

Reply:
लॉग्डइन done

Reply:
also this changes the context of the message, it should be:  हम एक `Greeting` कौम्पोनॅन्ट बनाएंगे जो यूजर ki लॉग्डइन sthiti ke anusar in dono main se 1 component dikhaega:

Reply:
हम एक Greeting कौम्पोनॅन्ट बनाएंगे जो यूजर की लॉग्डइन स्थिति  के अनुसार  इन दोनों में से एक कॉम्पोनेन्ट दिखायेगा

Reply:
done